libmcount/mcount.h

```diff
@@ -74,6 +74,11 @@ enum shmem_buffer_flags {
 	SHMEM_FL_RECORDING	= (1U << 2),
 };
 
+/* the methods used to tracing */
+#define METHOD_PRELOAD			1
+#define METHOD_ATTACH			2
+
```

Comment:
Why do you define them here?   Also please do not add blank lines unnecessarily.